docs/data/date-pickers/custom-field/custom-field.md

```diff
@@ -23,6 +23,17 @@ You can use the `fieldSeparator` slot to pass custom props to the `Typography` r
 
 {{"demo": "MultiInputFieldSeparatorSlotProps.js"}}
 
+### Use single input fields on range pickers [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan)
```

Comment:
I understand the idea of having this content at custom-field.md 
But I believe users will more intuitively find it on the main Date Range Picker page.
Any strong reason not to have it there?

Reply:
Once we have the new time range and date time range, would you put it in each of the component pages ?

I'm wondering if we should do both, one advanced example in the *custom-field.md* where we can give more detail in the future.
And one basic example on each component page with a link to *custom-field.md* if people want to learn more.

Reply:
I had the same feeling when looking through these changes and the updated doc. 🤔 


> I'm wondering if we should do both, one advanced example in the _custom-field.md_ where we can give more detail in the future.
> And one basic example on each component page with a link to _custom-field.md_ if people want to learn more.

I do think that this would make sense! 👍